examples/complex/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "stage-2", "stage-3"],
```

Comment:
Why exactly do we need Babel here? I suspect it's only being used for ES6 modules. We can assume a recent version of Node with native `async`. If we can remove Babel we would be saving on a lot of complexity and preventing some headaches.

I would suggest to remove Babel and switch to `require(...)` instead of `import`.